pages/linux/scan-build.md

```diff
@@ -0,0 +1,20 @@
+# scan-build
+
+> A command-line utility to run a static analyzer over a codebase as part of performing a regular build.
+> More information: <https://clang-analyzer.llvm.org/scan-build.html>.
+
+- Build and analyze the project in the current directory:
+
+`scan-build {{make}}`
+
+- Here is the general format for invoking scan-build:
+
+`scan-build [scan-build options] <command> [command options]`
```

Comment:
Perhaps this will read better and be more consistent with other pages?
```suggestion
- Invoke `scan-build`:

`scan-build [scan-build options] {{command}} [command options]`
```

This example should also be the last one on the page I think.